site/content/docs/main/troubleshooting.md

```diff
@@ -74,6 +74,37 @@ Here are some things to verify if you receive `SignatureDoesNotMatch` errors:
 Velero cannot resume backups that were interrupted. Backups stuck in the `InProgress` phase can be deleted with `kubectl delete backup <name> -n <velero-namespace>`.
 Backups in the `InProgress` phase have not uploaded any files to object storage.
 
+## Avoid Velero restarts due to Memory limits
+
+It may happen that the Velero process hits memory limits when backing up big system with a lot of resources.
```

Comment:
nit: i think you are missing 'a' in this sentence 'when backing up **a** big system with a lot of resources.'